lib/parameter_substitution/formatters/if_truthy.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class ParameterSubstitution::Formatters::IfTruthy < ParameterSubstitution::Formatters::Base
+  TRUTHY_VALUES = [true, "true", "t", 1, "1", "on", "yes"]
+
+  def self.description
+    "If the input is truthy (i.e. true, \"t\", 1, \"on\", \"yes\") then the input is replaced with the first argument. Otherwise, the input is replaced with the second argument."
```

Comment:
Should this do string interpolation of the `TRUTHY_VALUES` array so that the description is always updated with a new truthy value is added?

Reply:
Yes! Good idea.